collections/assets/dsko.json

```diff
@@ -1,13 +1,22 @@
 {
   "name": {
-    "Latn": "Sri Dasam Granth Sahib Translation"
+    "Latn": "Sri Dasam Granth Sahib Translation",
+    "Guru": "ਸ੍ਰੀ ਦਸਮ ਗ੍ਰੰਥ ਸਾਹਿਬ ਅਨੁਵਾਦ"
```

Comment:
Should these name fields be a language or script field?

Reply:
If you do language, you have to differentiate between gurmukhi punjabi and shahmukhi punjabi

The usage should probably be "src" as what it actually is regardless of language/script (could be multilingual/multiscript) and then it should be whatever we want to use for localisation (I'm assuming langauge)

Reply:
Instead of "src" could also be "data" or it could be separate fields, "title" as the actual and "titleLocalised" for the rest

Reply:
Note that `source.title` also exists, which is the original, non-localised title

Reply:
`name` is what we refer to it as, I suppose

Reply:
why not data.title? what is "source" meaning in the asset?

Reply:
meaning, I think "data" should stand for what these files actually are, "name" is a field we came up with for our own use

Reply:
> The usage should probably be "src" as what it actually is regardless of language/script (could be multilingual/multiscript) 

Bit of a tricky one - if it was a multiscript + multilingual (i.e. whatever) field, it makes it very hard for apps to do something meaningful with it. They should know whether it's a multiscript (not necessarily localisable) or multilingual (localisable) field

Reply:
do you want to chat about this on slack instead? LOL

Reply:
> why not data.title? what is "source" meaning in the asset?

The `source` field represents all citational / as-per-the-source data accurately

I think `data` is a little generic and the contents of the JSON file as you say is the data 😎

Reply:
I think "reference", "citation_data", or "citation" makes more sense then, as "source" is too generic and doesn't describe a scope well. My personal choice would be "reference" as what you're filling in here is used for citation references specifically

Reply:
I did have the same thought originally. But on second consideration, it seemed that a citation or reference is a derivative or use-case for this field, but the data it represents itself is the source of the asset itself. We could also just unwrap the fields from the `source` field entirely and drop it. Though I do think it's useful to have the 1-1 asset source data grouped together, since it distinguishes from other data/metadata we'd like to add ourselves

Reply:
Continued in Slack:
- Agreed on `reference` > `source`
- Only `title` is required
- ISO 15924 fields must require `Latn` only
- ISO 639 fields must require `en` only
- `name` must become a language/ISO 639 field, as the title only becomes meaningful in other languages when interpreted